Source/MVC6/Boilerplate.AspNetCore.TagHelpers/Twitter/TwitterCard.cs

```diff
@@ -58,31 +97,27 @@ public override string ToString()
         }
 
         /// <summary>
-        /// Appends a HTML-encoded string representing this instance to the <paramref name="stringBuilder"/> containing
-        /// the Twitter card meta tags.
+        /// Appends a HTML-encoded string representing this instance to the <paramref
+        /// name="stringBuilder"/> containing the Twitter card meta tags.
         /// </summary>
         /// <param name="stringBuilder">The string builder.</param>
         public virtual void ToString(StringBuilder stringBuilder)
         {
             this.Validate();
 
-            if (this.Type != TwitterCardType.Summary)
+            stringBuilder.AppendMetaNameContent("twitter:card", this.Type.ToTwitterString());
```

Comment:
I don't know if they used to not be required, but they have been since I researched this change.  You can confirm this on [Summary Large](https://dev.twitter.com/cards/types/summary-large-image) and [Player Cards](https://dev.twitter.com/cards/types/player#Reference) in the references.